src/core/load-fragment.c

```diff
@@ -2390,6 +2390,375 @@ int config_parse_documentation(const char *unit,
 }
 
 #ifdef HAVE_SECCOMP
+static int syscall_filter_parse_one(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                ExecContext *c,
+                bool invert,
+                const char *t,
+                bool warn) {
+        int r;
+
+        if (*t == '@') {
+                const char * const *s;
+                static const char * const syscall_sets[] = {
+                        /* Default list */
+                        "@Default",
+                        "execve\0"
+                        "exit\0"
+                        "exit_group\0"
+                        "rt_sigreturn\0"
+                        "sigreturn\0",
+
+                        /* Need superuser capabilities */
+                        "@SysAdmin",
+                        "acct\0"
+                        "adjtimex\0"
+                        "bdflush\0"
+                        "bpf\0"
+                        "chown32\0"
+                        "chown\0"
+                        "chroot\0"
+                        "create_module\0"
+                        "delete_module\0"
+                        "fchown32\0"
+                        "fchown\0"
+                        "fchownat\0"
+                        "finit_module\0"
+                        "init_module\0"
```

Comment:
@topimiettinen for Android stuff, I really don't know, but was thinking in case systemd is used to manage some base services which devices like that use, so you have the app abstract layer on top of that which can be used later to run Android apps... anyway this is a bit out of context, but maybe one day ! :-)